chat-commands.js

```diff
@@ -3254,11 +3251,11 @@ exports.commands = {
 
 		[code, stdout, stderr] = await exec(`git stash save --include-untracked "PS /updateserver autostash"`);
 		let stashedChanges = true;
-		if (code) throw new Error(`updateserver: Crash while stashing`);
+		if (code) throw new Error(`updateserver: Crash while stashing.`);
```

Comment:
This isn't a complete sentence; it should not end with a period

Reply:
It's also an Error, which as mentioned earlier doesn't have periods.